vocab/credentials/v2/vocabulary.yml

```diff
@@ -163,7 +171,7 @@ property:
 
   - id: verifiableCredential
     defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-verifiableCredential
-    label: Verifiable credential
+    label: Verifiable credential graph
```

Comment:
this was the change I was looking for.

Reply:
@iherman I was doing code reviews from my phone, its possible I missed this in your original PR.

The general comment was to address the labels and try to provide unique language that distinguished all the various "verifiable credential" labels.

It seems like my feedback has been addressed.